SharkCage/CageManager/CageLabeler.h

```diff
@@ -9,24 +9,74 @@ using namespace std;
 using namespace Gdiplus;
 #pragma comment (lib, "Gdiplus.lib")
 
-void DisplayTokenInCageWindow(HWND *hwnd);
-bool GetBottomFromMonitor(int &monitor_bottom);
+#include "../CageNetwork/NetworkManager.h"
+#include "../CageNetwork/MsgManager.h"
+#include "../CageNetwork/MsgService.h"
+#include "base64.h"
 
-HWND gotodesk_button;
+#include <optional>
+
+static bool DisplayTokenInCageWindow(HWND *hwnd);
+static bool GetBottomFromMonitor(int &monitor_bottom);
+static bool ShowExitButton(HWND &hwnd);
+static bool ShowConfigMetadata(HWND &hwnd);
+
+static HWND gotodesk_button;
+static HWND app_name_title;
+static HWND app_name_restart_button;
+static HWND additional_app_app_title;
+static HWND additional_app_restart_button;
+static HWND app_hash_text_title;
+static HWND app_hash_text;
+
+static std::wstring app_names;
+static std::wstring app_tokens;
+static std::wstring app_hashs;
```

Comment:
typo? trailing `s` at the end for each variable name

Reply:
The problem is that the `CALLBACK WndPro` can not be inside a class. Because of that, we have to movr the values (or the references) from the constructor to variables outside the class but we cannot use the same name like in a class context `this->foo = foo`
![screen shot 2018-06-10 at 1 07 59 am](https://user-images.githubusercontent.com/1786772/41196847-cc067a08-6c4a-11e8-9103-f2fee74ddaf4.png)

Maybe `m_app_name` or  `_app_name` is a better name?